src/CodeGen_Hexagon.cpp

```diff
@@ -1810,6 +1818,19 @@ void CodeGen_Hexagon::visit(const Call *op) {
                                 type_suffix(op->args[1], op->args[2], false),
                                 op->args);
             return;
+        } else if (op->is_intrinsic(Call::abs)) {
+            string v65orLater_suffix = "";
+            internal_assert(op->args.size() == 1);
+            Type ty = op->args[0].type();
+            if (ty.is_vector() && ty.is_int() && (ty.bits() == 8)) {
+                if (target.features_any_of({Target::HVX_v65, Target::HVX_v66})) {
+                    v65orLater_suffix = "_v65";
+                } else {
+                    internal_error << "vabs(i8) not supported for this target feature\n";
```

Comment:
When not supported, this should just fall through to the base class, which will just implement it with a select.

Reply:
You mean CodeGen_Posix::visit(op) , Right?

Reply:
Yes, but this code should simply only handle things it wants to handle and return early. Anything it can't handle, it should just ignore. CodeGen_Posix::visit(op) is called at the end of this function if nothing else handles the op first.